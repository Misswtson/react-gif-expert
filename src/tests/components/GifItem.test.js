import { render } from "@testing-library/react";
import { GifItem } from "../../components/GifItem";

describe('test en <GifItem/>', () => {
    const title = 'Darth Wader';
    const url = 'https://star-wars.com/darthwader.jpg';
    
    test('debe hacer match con el snapshot', () => { 
    const { container } = render( <GifItem title={ title } url={ url }/>);
    expect( container ).toMatchSnapshot();
});

});
