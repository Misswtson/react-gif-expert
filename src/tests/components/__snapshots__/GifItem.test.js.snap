// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test en <GifItem/> debe hacer match con el snapshot 1`] = `
<div>
  <div
    class="card"
  >
    <img
      alt="Darth Wader"
      src="https://star-wars.com/darthwader.jpg"
    />
    <p>
      Darth Wader
    </p>
  </div>
</div>
`;
